Makefile

```diff
@@ -144,3 +144,25 @@ create-metrics-svc:
 .PHONY: delete-metrics-svc
 delete-metrics-svc:
 	kubectl delete -f deploy/example/metrics/csi-blob-controller-svc.yaml --ignore-not-found
+
+# compile .proto file to go output
+.PHONY: gen-proto
```

Comment:
what's this for? is there any other place using `make gen-proto`?

Reply:
every we make changes to `pkg/blobfuse-proxy/proto/*.proto` files we should run this command. so that it will generate the files `pkg/blobfuse-proxy/pb/*.go`

Reply:
could you add a README.md file in `pkg/blobfuse-proxy` dir about how to make changes, build binary, and build testing blobfuse-proxy image, thanks.

Reply:
done